src/Configuration/Properties/Checkbox.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration\Properties;
+
+class Checkbox extends Property
+{
+    public function __construct(
+        string $name,
+        string $description,
+        bool $isRequired = false,
+        string $hint = '',
```

Comment:
could be nullable (same for the other similar classes):
```suggestion
        ?string $hint = null,
```